src/mock/oppfolging.ts

```diff
@@ -0,0 +1,35 @@
+const oppfPerioder: any[] = [
```

Comment:
Kan vi lage sånn at vi kan skrive `Oppfolgingsperiode[]` her?
Alltid kjekt å bruke type-definisjonene der det er mulig. :)

Reply:
Husker ikke helt hvorfor jeg satte den til any[], men når jeg bruker`Oppfolgingsperiode[]` får jeg typescript error og blant annet dette: ` Property 'oppfolgingsPerioder' is incompatible with index signature.
      Type 'OppfolgingData[]' is not assignable to type 'JSONValue'.`

Reply:
Det er mock get som forventer å få json object. Kanskje man kan konvertere objektet og returnere det?